csharp/src/Microsoft.Azure.EventHubs.Processor/PartitionContext.cs

```diff
@@ -20,8 +18,20 @@ internal PartitionContext(EventProcessorHost host, string partitionId, string ev
             this.EventHubPath = eventHubPath;
             this.ConsumerGroupName = consumerGroupName;
             this.ThisLock = new object();
+            this.SequenceNumber = 0;
+            this.Offset = PartitionReceiver.StartOfStream;
         }
 
+        /// <summary>
+        /// Returns the current Offset of the partition.
+        /// </summary>
+        public string Offset { get; private set; }
+
+        /// <summary>
+        /// Returns the current sequence number of the partition.
```

Comment:
so is the right wording "Returns the current sequence number from the last checkpoint of the partition"?